rastervision/experiment/experiment_config.py

```diff
@@ -163,6 +164,10 @@ def __init__(self, prev=None):
 
     def validate(self):
 
+        backend = self.config.get('backend')
+        if not issubclass(type(backend), BackendConfig):
+            raise rv.ConfigError('Call "build()" on the backend.')
```

Comment:
Not calling  `build` on a `BackendConfigBuilder` is one way for the types not to line up - but not all the ways the type won't match. This error should be more like:

`'Backend set with "with_backend" needs to be of type BackendConfig, got {}'.format(type(backend))`

Reply:
yeah, that makes sense, thanks